dist/minified.json

```diff
@@ -1 +1 @@
e":"energy_class"},"energy_class_f":{"code":"energy_class_f","labels":{"en_US":"F"},"attribute":"energy_class"},"energy_class_g":{"code":"energy_class_g","labels":{"en_US":"G"},"attribute":"energy_class"},"operating_system_windows":{"code":"operating_system_windows","labels":{"en_US":"Windows"},"attribute":"operating_system"},"operating_system_apple":{"code":"operating_system_apple","labels":{"en_US":"Apple"},"attribute":"operating_system"},"operating_system_android":{"code":"operating_system_android","labels":{"en_US":"Android"},"attribute":"operating_system"},"picture_quality_8k":{"code":"picture_quality_8k","labels":{"en_US":"8K"},"attribute":"picture_quality"},"picture_quality_4k":{"code":"picture_quality_4k","labels":{"en_US":"4K"},"attribute":"picture_quality"},"picture_quality_hd":{"code":"picture_quality_hd","labels":{"en_US":"HD"},"attribute":"picture_quality"},"picture_quality_sd":{"code":"picture_quality_sd","labels":{"en_US":"SD"},"attribute":"picture_quality"},"picture_quality_full_hd":{"code":"picture_quality_full_hd","labels":{"en_US":"Full HD"},"attribute":"picture_quality"},"picture_quality_ultra_hd":{"code":"picture_quality_ultra_hd","labels":{"en_US":"Ultra HD"},"attribute":"picture_quality"},"processor_qualcomm_snapdragon":{"code":"processor_qualcomm_snapdragon","labels":{"en_US":"Qualcomm Snapdragon"},"attribute":"processor"},"processor_apple_a_series":{"code":"processor_apple_a_series","labels":{"en_US":"Apple A-series"},"attribute":"processor"},"processor_samsung_exynos":{"code":"processor_samsung_exynos","labels":{"en_US":"Samsung Exynos"},"attribute":"processor"},"processor_mediatek_dimensity":{"code":"processor_mediatek_dimensity","labels":{"en_US":"MediaTek Dimensity"},"attribute":"processor"},"processor_huawei_kirin":{"code":"processor_huawei_kirin","labels":{"en_US":"Huawei Kirin"},"attribute":"processor"},"processor_intel_atom":{"code":"processor_intel_atom","labels":{"en_US":"Intel Atom"},"attribute":"processor"},"processor_hisilicon_kirin":{"code":"processor_hisilicon_kirin","labels":{"en_US":"HiSilicon Kirin"},"attribute":"processor"},"processor_nvidia_tegra":{"code":"processor_nvidia_tegra","labels":{"en_US":"NVIDIA Tegra"},"attribute":"processor"},"processor_amd_ryzen_mobile":{"code":"processor_amd_ryzen_mobile","labels":{"en_US":"AMD Ryzen Mobile"},"attribute":"processor"},"processor_mediatek_helio":{"code":"processor_mediatek_helio","labels":{"en_US":"MediaTek Helio"},"attribute":"processor"},"processor_dual_core":{"code":"processor_dual_core","labels":{"en_US":"Dual Core"},"attribute":"processor"},"processor_mediatek":{"code":"processor_mediatek","labels":{"en_US":"MediaTek"},"attribute":"processor"}}},"snacks":{"code":"snacks","labels":{"en_US":"Snacks"},"description":{"en_US":"Satisfy your cravings with our 'Snacks' family template - a delicious assortment of pre-built attributes to effortlessly manage your favorite treats, making snack time a delightful experience."},"attribute_as_main_media":"main_image",
```

Comment:
Please don't commit it, there's a workflow which is triggered to do it on merge on min 👍🏻